src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepositorySpec.scala

```diff
@@ -67,12 +90,12 @@ class FileMetadataRepositorySpec extends AnyFlatSpec with ScalaFutures with Matc
     TestUtils.createFile(UUID.fromString(fileTwoId), consignmentOne)
     TestUtils.createFile(UUID.fromString(fileThreeId), consignmentTwo)
 
-    TestUtils.addFileProperty(propertyId, SHA256ServerSideChecksum)
+    TestUtils.addFileProperty(SHA256ServerSideChecksum)
 
 //  Then need to add data to the FileMetadata repository for these files
-    TestUtils.addFileMetadata(metadataOneId, fileOneId, propertyId)
-    TestUtils.addFileMetadata(metadataTwoId, fileTwoId, propertyId)
-    TestUtils.addFileMetadata(metadataThreeId, fileThreeId, propertyId)
+    TestUtils.addFileMetadata(metadataOneId, fileOneId, SHA256ServerSideChecksum)
+    TestUtils.addFileMetadata(metadataTwoId, fileTwoId, SHA256ServerSideChecksum)
+    TestUtils.addFileMetadata(metadataThreeId, fileThreeId, SHA256ServerSideChecksum)
 
     val fileMetadataFiles = fileMetadataRepository.countProcessedChecksumInConsignment(consignmentOne).futureValue
```

Comment:
Could check the metadata for both consignments created. Only checking `consignmentOne` at the moment

Reply:
I've added in an assertion for consignment two.